CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+# 1.0.0
+
+- VSCode centric Connection settings. . Thanks @crawr, @jellypuno
+    - Credential prompting in profiles and favorite . Thanks @crawr, @jellypuno
+- Dataset and Dataset member copy and renaming function. Thanks @CForrest97
+- Theia support including documentation 
```

Comment:
Unnecessary trailing space

Reply:
Whoops sorry guys. Should have run the linter.